### DISCUSSION TITLE
Several issues should be reopened

### DISCUSSION THREAD

**Discussion by tmlmt** [primefaces/community#665] [PrimeVue]
Hi team, these issues have received comments confirming the bug is still there, or reappeared, although the issues have been closed. Could you help reopening them, or shall new issues be created?
- https://github.com/primefaces/primevue/issues/4552
- https://github.com/primefaces/primevue/issues/4210
- https://github.com/primefaces/primevue/issues/1723

**Comment by tugcekucukoglu** [comment]
You can open an issue if you think it is a bug. If there is an open issue, trying to follow both issue tracking and discussions topic at the same time will become a bit complicated.

**Comment by tmlmt** [comment]
OK. I've created a new issue for the first one. I will create new issues for the two others when I can figure out a reproducer.
